modules/vpc/outputs.tf

```diff
@@ -1,2 +1,7 @@
-# Outputs
+output "iam_policy_developer_arn" {
```

Comment:
Did we document in CHANGELOG why we needed these for VPC?

Reply:
Have now!